sw/device/lib/testing/spi_flash_testutils.c

```diff
@@ -85,29 +86,65 @@ void spi_flash_testutils_read_sfdp(dif_spi_host_t *spih, uint32_t address,
                                         ARRAYSIZE(transaction)));
 }
 
+status_t spi_flash_testutils_read_status(dif_spi_host_t *spih, uint8_t opcode,
+                                         size_t length) {
+  CHECK(spih != NULL);
```

Comment:
Maybe `TRY_CHECK`?

Reply:
Lets defer that for a follow-on PR that cleans up spi_flash_testutils with respect to `status_t`.